Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM kkarczmarczyk/node-yarn:6.9
+FROM node:12 as builder
 
-RUN mkdir /plugins
-COPY ./.babelrc /plugins
-COPY ./package.json /plugins
-COPY ./server.js /plugins
-COPY ./app/ /plugins/app/
-COPY ./public/ /plugins/public
-COPY ./views/ /plugins/views
-COPY ./webpack /plugins/webpack
-COPY ./yarn.lock /plugins
-WORKDIR /plugins
+ARG DISABLE_SEARCH_ENGINE
+ARG GET_CONTENT
+ARG GATSBY_CONFIG_SITE_METADATA__URL
+ARG GATSBY_CONFIG_SITE_METADATA__SITE_URL
+WORKDIR /usr/src/app
 
-RUN yarn
+ADD package.json yarn.lock ./
+RUN yarn install
+ADD . .
+RUN yarn build
+RUN test -e public/index.html || exit 1
+RUN yarn lint
+RUN yarn test
 
-CMD yarn server
-
-EXPOSE 5000
+# FROM nginx:1.17.6
```

Comment:
is the intention to use nginx image? what's the reason for this comment being here?

Reply:
left behind probably. the gatsby image is way faster than stock nginx, and i still think cdn > docker.